### PR TITLE
Register.lic updates

### PR DESCRIPTION
**Pull request by BinuDR**
Added the ability to label deeds with the register script.

You can either:
`;register label #` to label a specific page
Or:
`;register label all` to label all unlabeled pages in your register.
This will not overwrite any previously labeled pages.
You can do that with the option to label a specific page if needed.

### CODE REVIEW

File: register.lic [hunk 1]
```diff
@@ -16,27 +18,126 @@ class Register
...
+  end
+
+  def get_item(item)
+    unless DRCI.get_item(item, @container)
+      case bput("get my #{item}", /You get/, /You are already holding that/, /What were you referring to/, /You need a free hand/)
+      when /What were you referring to/
+        message("Can't find your #{item}!")
         exit
+      when /You need a free hand/
+        message("Your hands are full!")
```
Comment (rpherbig) [inline register.lic:63]: Should we really continue in this case?

File: register.lic [hunk 2]
```diff
@@ -16,27 +18,126 @@ class Register
...
+  end
+
+  def label_page
+    quality, volume, pure, mix, type = nil
+    fput('read my register')
+    loop do
+      line = get
+      case line
+      when /Metallurgical Properties/
+        type = 'metal' #if line =~ /Metallurgical Properties/
```
Comment (rpherbig) [inline register.lic:98]: Can you delete the commented out code?

File: register.lic [hunk 3]
```diff
@@ -16,27 +18,126 @@ class Register
...
+        volume = Regexp.last_match(1).to_i
+      when /This ingot is certified pure/
+        pure = true
+      when /The metal appears to be composed of: (.*)\.$/
+        mix = Regexp.last_match(1)
+      when /^(?:You think it will still take|You think the item is ready for reclaiming)/
+        break
+      end
+    end
+    # mix = sort_mix(mix) if mix
```
Comment (rpherbig) [inline register.lic:111]: Can you delete the commented out code?

**Comment by BinuDR** [comment]
All 3 issues taken care of.

**Pull request merged**